### ISSUE TITLE
Add breathing room to H2 spacing

### ISSUE THREAD

**Issue by valgrimm** [mdn/developer-portal#1382]
AC: Add 1 line of spacing above H2 titles to match design

(otherwise this has to be added manually with a < br >, as has been done on the Hubs page)

**Comment by stevejalim** [comment]
Can you add a screenshot of what you want it to look like and on which page, please? (ie am assuming you're talking about a content page/rich text, but need to check as we have a lot of other H2s in the site)

**Comment by valgrimm** [comment]
The top image below shows a page where I have added two brs between the image and the heading. The bottom image below shows one where I have removed those brs.

Spacing should be as in the top image by default. 

![withbrbr](https://user-images.githubusercontent.com/56933147/81583121-2ce63400-93b1-11ea-9ffa-95652227738e.PNG)
![withoutbr](https://user-images.githubusercontent.com/56933147/81583126-2f488e00-93b1-11ea-8f7a-7ad285538272.PNG)
